lumigator/python/mzai/backend/backend/services/experiments.py

```diff
@@ -122,13 +122,17 @@ def create_experiment(self, request: ExperimentCreate) -> ExperimentResponse:
             "--config": config_template.format(**config_params),
         }
 
+        # as our command relies on libgomp which gives issues on aarch64, we
+        # preload it with LD_PRELOAD_PREFIX (see settings.py for details)
+        eval_command = f"{settings.LD_PRELOAD_PREFIX} python -m evaluator evaluate huggingface"
```

Comment:
Add a note as to why we're setting here and not in env_vars (presumably because VIRTUAL_ENV hasn't been resolved in env vars yet) 

Also, this should only get set in certain circumstances (only on mac) not in all environments. This was implicitly handled before by only setting it the local docker compose

Reply:
Thanks!
- added rationale here
- added architecture-specific setting of the variable (in settings.py for readability, with explanation)